static/app/views/settings/dynamicSampling/dynamicSampling.tsx

```diff
@@ -25,6 +32,8 @@ export function DynamicSampling() {
   const organization = useOrganization();
   const {hasAccess} = useAccess({access: ['org:write']});
 
+  const [period, setPeriod] = useState<'24h' | '30d'>('24h');
```

Comment:
nit: we are using `24 | 30d` in 2x different files...what about we make it reusable?